ArkKit/Ark.swift

```diff
@@ -14,9 +14,14 @@ class Ark {
     let rootView: UINavigationController
     let eventManager = ArkEventManager()
     let ecsManager = ArkECS()
+    let gameScene: AbstractArkGameScene // Ideas for naming this please, maybe physics scene but it sounds weird
 
     init(rootView: UINavigationController) {
         self.rootView = rootView
+        // TODO: Change to take in the game size instead of the root nav controller
+        // Currently the simulator is defined to use the Sprite Kit simulator
+        // In the future, we can look at exposing this in the blueprint
+        self.gameScene = SKGameScene(size: rootView.view.frame.size)
```

Comment:
I think this might not work, because the `rootView.view.frame.size` is only set after the lifecycle method `viewDidSubviewsLayout` is called, as far as I understand. 

We could move this setup of the size as a delegate instead? Something like

```swift
rootView.gameSceneDelegate = SKGameSceneDelegate()
```

Reply:
Sounds good